action.yml

```diff
@@ -10,7 +10,11 @@ inputs:
     required: true
   number:
     description: 'Pull Request number'
-    required: true
+    required: false
+  async:
```

Comment:
I think a better name would be `waitForPullRequestUpdated`, which defaults to `true`. Iiuc this doesn't really have anything to do with synchronous vs asynchronous

Reply:
Updated to waitForPullRequestUpdated. I'll retest then resolve this comment.